ml-agents/mlagents/trainers/stats.py

```diff
@@ -58,6 +55,22 @@ def aggregated_value(self):
         else:
             return self.mean
 
+    @property
+    def mean(self):
+        return np.mean(self.full_dist)
+
+    @property
+    def std(self):
+        return np.std(self.full_dist)
+
+    @property
+    def num(self):
+        return len(self.full_dist)
```

Comment:
`StatsSummary.empty().num` will return 1 now.

Reply:
My first thought is to use an empty list for `full_dist` when calling `empty()`. That means the other stats will return `NaN` which is technically more correct than returning `0`, thought may break something else. Do you have a different preference?

Reply:
Only other thought is to do e.g. `np.mean(self.full_dist) if self.full_dist) else 0.0` which is basically the old behavior, and same for `np.std` (`np.sum` is safe)

Reply:
But I think this is fine too.